charts/spire/charts/spire-agent/values.yaml

```diff
@@ -206,8 +206,14 @@ workloadAttestors:
   k8s:
     ## @param workloadAttestors.k8s.enabled Enables the Kubernetes workload attestor
     enabled: true
-    ## @param workloadAttestors.k8s.skipKubeletVerification If true, kubelet certificate verification is skipped
-    skipKubeletVerification: true
+    verification:
+      ## @param workloadAttestors.k8s.verification.type What kind of verification to do against kubelet. Valid options are [hostCert, apiServerCA, skip]
+      type: hostCert
```

Comment:
@kfox1111 I like this changeset but Im concerned about this verification not working with some environments. Would you be ok with setting this to `skip` as the default for a release or two? Until we can determine what environments this check will or won't work for.

Reply:
Hmm... We could... But we're going to need to rip the bandaid off at some point. I'd also like to have some time under our belt with it enabled before we cut a 1.0. So, do we think we're say, 4+ major releases out from 1.0? I was thinking we are pretty close to being able to do a 1.0. Like, maybe we could have 1.0 around kubecon time?

Reply:
Thanks for setting the default to `skip`